content/millennium/r4/foundation/security/provenance.md

```diff
@@ -9,31 +9,38 @@ title: Provenance | R4 API
 
 ## Overview
 
-The Provenance resource tracks the source of external data and is used to assess the trustworthiness and reliability of a resource. Sources can be an organization author or organization transmitter.  Entities are Continuity of Care DocumentReferences (CCDs).
+The Provenance resource tracks the source of external information and is used to assess the trustworthiness and reliability of a resource. Sources can be an organization author or organization transmitter. The `Target` field references a Resource that this Provenance entry is related to. Entities are Continuity of Care Documents (CCDs) generated through the [DocumentReference] resource.
```

Comment:
since it references the field directly, it should match the case of the field.
```suggestion
The Provenance resource tracks the source of external information and is used to assess the trustworthiness and reliability of a resource. Sources can be an organization author or organization transmitter. The `target` field references a resource that this Provenance entry is related to. Entities are Continuity of Care Documents (CCDs) generated through the [DocumentReference] resource.
```